### ISSUE TITLE
Companion app cannot be installed on older devices

### ISSUE THREAD

**Issue by FinnKr** [jwlilly/Android-Accessibility-Inspector-App#3]
Steps to reproduce:
1. Connect android device with Android 10 (API level 29) or lower
2. Open Android Accessibility Inspector
3. Tap on "Install companion app" in menu

Actual result:
* Error is displayed: `Command failed: "/Applications/android-accessibility-inspector.app/Contents/Resources/app.asar.unpacked/src/assets/adb" install -r -d "/Applications/android-accessibility-inspector.app/Contents/Resources/app.asar.unpacked/src/assets/app-release.apk"
adb: failed to install /Applications/android-accessibility-inspector.app/Contents/Resources/app.asar.unpacked/src/assets/app-release.apk: Failure [INSTALL_FAILED_OLDER_SDK: Failed parse during installPackageLI: /data/app/vmdl1728276589.tmp/base.apk (at Binary XML file line #7): Requires newer sdk version #30 (current version is #29)]`

Expected result:
* App will be installed on target device

Possible solution:
* Build the bundled companion app with minSdkVersion 29 or lower (if this is possible)

**Comment by FinnKr** [comment]
This might not be possible if the app relies on APIs unavailable to sdk version 29 or lower.

**Comment by jwlilly** [comment]
@FinnKr The only thing holding me back was the screenshot feature but I've moved that to an adb command. If I send you an APK to test, do you know how to sideload it onto the device/virtual device?

**Comment by FinnKr** [comment]
> @FinnKr The only thing holding me back was the screenshot feature but I've moved that to an adb command. If I send you an APK to test, do you know how to sideload it onto the device/virtual device?

Yes I can do that.

**Comment by jwlilly** [comment]
Try this one. The minSdk was changed to 28: https://github.com/jwlilly/Accessibility-Inspector-Service/releases/tag/1.2-alpha

After installing, you shouldn't need to install the companion app from the toolbar. On first launch, you may need to manually open the app and allow notifications and then go into accessibility settings and enable the service there. Some devices are picky on the first run.

**Comment by FinnKr** [comment]
> Try this one. The minSdk was changed to 28: https://github.com/jwlilly/Accessibility-Inspector-Service/releases/tag/1.2-alpha
> 
> After installing, you shouldn't need to install the companion app from the toolbar. On first launch, you may need to manually open the app and allow notifications and then go into accessibility settings and enable the service there. Some devices are picky on the first run.

Thank you. This version worked on my device with Android 10 (API 29).

**Comment by jwlilly** [comment]
You're welcome. I'll leave this issue open so I can update this repo with the latest companion version this weekend

**Comment by jwlilly** [comment]
Fixed with version 2.0
